conference/react-js/src/containers/Talks/talks.scss

```diff
@@ -16,7 +16,7 @@
 		}
 
 	@media ($mnw: $xmdmx) {
-		height: calc(100vh - 200px);
+		height: calc(100vh - 375px);
```

Comment:
Can we use variable for "375px" ? because you changed 200px => 375px in multiple place